README.md

```diff
@@ -80,14 +80,11 @@ $ apt-get install manif
 ```terminal
 $ git clone https://github.com/artivis/manif.git
 $ cd manif && mkdir build && cd build
-$ cmake ..
-$ make
-```
-###### To build also examples/tests
-```terminal
 $ cmake -DBUILD_TESTING=ON -DBUILD_EXAMPLES=ON ..
 $ make
```

Comment:
It appears 'make' alone does nothing no? Or is it to make tests and examples?

Reply:
Both tests and examples build are enabled thus `make` should build both.